scripts/report_scripts/index.Rmd

```diff
@@ -108,8 +108,12 @@ approved_mut_plot <- df_mut %>% filter( samplename  %in% good_samples.df$samplen
 # pool the samples per day, discard locations
 weights <- read.csv(params$overviewQC, header = TRUE, check.names = FALSE)
 weights <- weights %>% dplyr::select( c(samplename, total_reads))
-approved_var_plot_location_pooled <- pool_by_weighted_mean(approved_var_plot, weights, "day_location")
 
+if (nrow(approved_var_plot) > 0) { # check if filtered dataframe has actual values
```

Comment:
I agree with the later comment in the code that says that we should just skip the whole snippet when there are no variants.  Duplicating this condition check everywhere is not great.